lib/ansible/module_utils/cnos.py

```diff
@@ -3160,13 +3175,16 @@ def checkOutputForError(output):
         index = output.lower().find("invalid")
         startIndex = index + 8
         if(index == -1):
-            index = output.lower().find("incorrect")
+            index = output.lower().find("Cannot be enabled in L2 Interface")
             startIndex = index + 9
```

Comment:
Is this maybe meant to be 33, and the one attached to `"incorrect"` be 9 instead?

Reply:
Hi Qalthos,
Good catch, Yes its a bug. I have swapped the position of "incorrect" and "Cannot be enabled in L2 Interface" to contain the pep8 requirements but missed to swap the index positions. What should i do in this regard? Do I have to pull the change again or change this as a bug fix in another pull ?

Reply:
Or can u swap it for me ?

Reply:
The PR is still open, just add another commit to the branch that this change is attached to and push it again, or even edit the file through the web editor in the PR itself, which essentially does the same thing